test/Misc/serialized-diagnostics-batch-mode-nonprimary-errors.swift

```diff
@@ -1,16 +0,0 @@
-// Ensure that an error in a non-primary causes an error in the errorless primary.
```

Comment:
I can't tell if this one is still relevant or not.

Reply:
I don't think it is still needed because it checks for:
1. The absence of "compilation stopped by errors in other files", which is a string which is not in the compiler at all anymore, AFAICT.
2. The empty.dia file existing and being empty, but this same check is in the remaining test, `serialized-diagnostics-batch-mode-nonprimary-errors-only.swift`.

Summary: one predicate is meaningless today because the code is no longer in the compiler, and the other is covered by the remaining test.

Reply:
I guess I feel like it's still a slightly different configuration even if it doesn't test any new code paths. But I'll defer to you on this.

Reply:
Thanks. You have a point, but I think I'm confident that this test doesn't carry it's weight.